### DISCUSSION TITLE
Taskbar by @farmerbb never updated 2 years ago

### DISCUSSION THREAD

**Discussion by TruongTanLuc201270** [community/community#125190] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

The lastest release 6.2, December 14 2021 but not updated 2 years ago. This version about a cannot change start button image, I put png image and apply doesn't work.
Taskbar never answer issues but I think @farmerbb still active but he was abandoned, I think Taskbar will removed due no longer development.

**Comment by ebndev** [comment]
Hi @TruongTanLuc201270,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by ebndev** [comment]
Hi again @TruongTanLuc201270!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Comment by TruongTanLuc201270** [comment]
Taskbar never fix issues and users not requested by @farmerbb.
I think taskbar need archive because no longer develop.

**Reply by ebndev** [reply]
Thanks for taking the time to respond! When you wanted to send feedback, did you manually go to GitHub Discussions or were you re-directed from somewhere on their profile / repo?

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
